kiwi/bootloader/config/grub2.py

```diff
@@ -64,8 +64,8 @@ def post_init(self, custom_args):
         self.firmware = FirmWare(
             self.xml_state
         )
-
-        self.hybrid_boot = True
+        if arch == 'x86_64':
+            self.hybrid_boot = True
```

Comment:
openSUSE 13.2 ist still also in the queue. But even without 32bit support it would be nice to not have this if condition, it would also make your life easier with the test coverage